lib/puppet/type/keycloak_identity_provider.rb

```diff
@@ -258,7 +276,7 @@ def self.title_patterns
     if self[:realm].nil?
       raise Puppet::Error, 'realm is required'
     end
-    if self[:ensure].to_s == 'present' && self[:provider_id] == 'oidc'
+    if self[:ensure].to_s == 'present' && (self[:provider_id] == 'oidc' || self[:provider_id] == 'keycloak-oidc')
```

Comment:
Might be shorter to say `['oidc', 'keycloak-oidc'].include?(self[:provider_id])` and if another provider_id is added it would be simpler to add new option without having extremely log line.